src/BizHawk.Emulation.Cores/Computers/DOS/DOSBox.ISettable.cs

```diff
@@ -8,49 +8,88 @@ namespace BizHawk.Emulation.Cores.Computers.DOS
 {
 	public partial class DOSBox : ISettable<object, DOSBox.SyncSettings>
 	{
+		public const int MAX_MEMORY_SIZE_MB = 256;
+
 		public enum ConfigurationPreset
 		{
-			[Display(Name = "[1981] IBM XT 5150 (4.77Mhz, 256kb RAM, Monochrome, PC Speaker)")]
+			[Display(Name = "[1981] IBM XT 5150 (4.77Mhz, 8086, 256kb RAM, Monochrome, PC Speaker)")]

```

Comment:
Here and in other places, does `b` mean bits or bytes? For the latter it's better to use uppercase `B` to clearly mean bytes.

Reply:
fixed